### PR TITLE
Remove blocking of dates.

### PR DESCRIPTION
**Pull request by knight-bubble**
https://community.openproject.org/work_packages/19857
This PR depends on #2960.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://hirtie-maxim-fix-19857-datepicker-date-blocking.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://hirtie-maxim-fix-19857-datepicker-date-blocking.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by floriank** [comment]
@hirtie-maxim this might have to be rebased, as #2960 has been merged

**Comment by floriank** [comment]
@hirtie-maxim specs fail consistently (restarted twice)

**Comment by floriank** [comment]
This looks good functionality wise, i can move the dates as specified, but I of course get an error from the API saying that the due date must be greater than the start date (they cannot be the same).

There is no acceptance criteria specified for this in the corresponding Epic. @NobodysNightmare do you know anything about this change? I will ask @lindenthal for more detailed behaviour-specs if necessary.

**Comment by floriank** [comment]
This does not work currently.

It seems that the dates sent are incorrect, so the API will throw a validation error, while the frontend shows start and due date with the same values. The API call seems to use the old values:

screenshot from 2015-05-12 11 57 40

Please investigate.

**Comment by floriank** [comment]
:+1: functionality

File: frontend/tests/integration/specs/work-packages/details-pane/details-pane-helper.js [hunk 1]
```diff
@@ -68,6 +68,7 @@ function behaveLikeEmbeddedDropdown(name, correctValue) {
           expect(
             editor
             .$('.select2-container').isDisplayed()
+            .isDisplayed()
```
Comment (floriank) [inline frontend/tests/integration/specs/work-packages/details-pane/details-pane-helper.js]: I think there is my favourite rebase-error once again :wink:

**Pull request merged**